pinot-servicemanager/Dockerfile

```diff
@@ -18,29 +16,29 @@ RUN mkdir classes && cd classes && \
   (for JAR in ../libs/*; do jar -xf $JAR; done) && \
   cd .. && rm -rf libs
 
+ADD install /tmp/install
+RUN /tmp/install && rm /tmp/install
+
 # Share the same base image to reduce layers used in testing
 FROM hypertrace/java:11
-LABEL maintainer="Hypertrace https://www.hypertrace.org/"
+LABEL MAINTAINER Hypertrace "https://www.hypertrace.org/"
 
 # Add HEALTHCHECK and ENTRYPOINT scripts into the default search path
 COPY docker-bin/* /usr/local/bin/
 
-ENV PINOT_HOME=/opt/pinot
-ENV PINOT_VAR=/var/pinot
-ENV JAVA_OPTS="-Xms256M -Xmx512M -XX:MaxDirectMemorySize=96M -XX:+ExitOnOutOfMemoryError"
-
-WORKDIR ${PINOT_HOME}
+# All content including binaries and logs write under WORKDIR
+ARG USER=pinot
+WORKDIR /opt/pinot
 
 # Ensure the process doesn't run as root
-ARG USER=pinot
-RUN adduser -g '' -h ${PWD} -D ${USER} && \
-  mkdir -p ${PINOT_VAR}/server/data ${PINOT_VAR}/controller/data && \
-  chown -R ${USER} ${PINOT_VAR}
+RUN adduser -g '' -h ${PWD} -D ${USER}
 USER ${USER}
 
 # Copy binaries and config we installed earlier
 COPY --from=install --chown=${USER} /install .
 
+ENV JAVA_OPTS="-Xms256M -Xmx512M -XX:MaxDirectMemorySize=96M -XX:+ExitOnOutOfMemoryError"
```

Comment:
I know you are just reshuffling this lines but I believe knowing why this settings and not others is good (e.g. why `MaxDirectMemorySize=96M` and not `128M`). At least for non javaers.

Reply:
I don't really have the answer to that one. I can remove that arg :D I do know the min heap needs to be 256m

Reply:
removed